src/main/java/software/amazon/qldb/QldbDriverBuilder.java

```diff
@@ -168,4 +169,6 @@
      * @return This builder object.
      */
     QldbDriverBuilder readAhead(int readAhead, ExecutorService executorService);
+
+    QldbDriverBuilder httpClientBuilder(SdkHttpClient.Builder httpClientBuilder);
```

Comment:
Nit: Why is this prefixed with `SdkHttpClient`?

Reply:
It's using `SdkHttpClient.Builder` defined in https://github.com/aws/aws-sdk-java-v2/blob/58c7300cd3d32f5c91c8f9e8b4f774826153321b/http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpClient.java#L62